### PR TITLE
[4.0] RavenDB-10732 - Memory Computation in Studio

### PR DESCRIPTION
**Pull request by grisha-kotler**
http://issues.hibernatingrhinos.com/issue/RavenDB-10732

### CODE REVIEW

File: src/Raven.Studio/typescript/models/resources/serverDashboard/machineResources.ts [hunk 1]
```diff
@@ -31,29 +37,71 @@ class machineResources {
...
+                                <br /><strong>Commit Threshold: ${this.sizeFormatter(commitThreshold)}</strong><br />`;
+                }  
+            }
+
+            return `${tooltip}</div>`;
         });
     }
-    
+
+    private removeSpaces(str: string): string {
```
Comment (ml054) [inline src/Raven.Studio/typescript/models/resources/serverDashboard/machineResources.ts]: is it used?

Comment (grisha-kotler) [reply src/Raven.Studio/typescript/models/resources/serverDashboard/machineResources.ts]: no. removed.

File: src/Raven.Studio/wwwroot/Content/css/bootstrap/variables.less [hunk 2]
```diff
@@ -560,7 +560,7 @@
 //##
 
 //** Tooltip max width
-@tooltip-max-width:           200px;
+@tooltip-max-width:           215px;
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/Content/css/bootstrap/variables.less]: can we scope this to dashboard only?

Comment (ml054) [reply src/Raven.Studio/wwwroot/Content/css/bootstrap/variables.less]: css selector:
#dashboard-container .tooltip

Comment (grisha-kotler) [reply src/Raven.Studio/wwwroot/Content/css/bootstrap/variables.less]: done

**Review by grisha-kotler — COMMENTED** [review]
Can be merged

**Pull request merged**